services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.tsx

```diff
@@ -131,7 +131,6 @@ export const EnvironmentListItem: React.FC<{
                     groupNameOverride={undefined}
                     numberEnvsDeployed={undefined}
                     numberEnvsInGroup={undefined}
-                    withEnvLocks={true}
```

Comment:
the `withEnvLocks` property was removed already from the environment chips
this is just additional cleanup